Adyen/UI/Form/Items/Label/UILabel.swift

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright (c) 2020 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+extension UILabel: FormItem, AnyFormItemView {
+    public var delegate: FormItemViewDelegate? {
+        get {
+            objc_getAssociatedObject(self, &UIViewAssociatedKeys.delegate) as? FormItemViewDelegate
+        }
+        set {
+            objc_setAssociatedObject(self,
+                                     &UIViewAssociatedKeys.delegate,
+                                     newValue,
+                                     objc_AssociationPolicy.OBJC_ASSOCIATION_RETAIN_NONATOMIC)
```

Comment:
policy should be `objc_AssociationPolicy.OBJC_ASSOCIATION_ASSIGN`, its a delegate, right?

Reply:
good point!